### ISSUE TITLE
[Very Minor Issue] CSV output decimal formatting.

### ISSUE THREAD

**Issue by Mitsuma** [rlguy/Blender-FLIP-Fluids-Beta#225]
Not sure if this is even worth mentioning but I thought why not. **This only affects users who have use a different decimal symbol.** You can work around it, so this does not need proper fixing I suppose, only if it is no big change.

The Statistic CSV file has times like total_time formatted with a dot. 
![excel_2018-03-28_15-16-57](https://user-images.githubusercontent.com/8568089/38034570-e0144746-32a2-11e8-8637-b72ed468508c.png)
So far so good, besides the odd long decimals after it.
Now the problem comes if you do not pay attention during conversion of the CSV into a table.

If you leave the formatting on default ("Standard") it will actually ignore the dot and it will turn it into a huge number. 
![dllhost_2018-03-28_16-17-12](https://user-images.githubusercontent.com/8568089/38034901-814bccf6-32a3-11e8-994d-ff939170c2e1.png)
But not only that, it will also make them of varying lentgh.  

**Workaround:**
Workaround is rather simple, you need to make sure the formatting for the table is set to "Text" not "Standard" in the promt during the table creation. (in Office)
![excel_2018-03-28_15-21-40](https://user-images.githubusercontent.com/8568089/38035019-b9fe348a-32a3-11e8-8c44-3084885358c4.png)
This will give you the proper format with a dot. Left side in that image above is the "standard" format, it actually also adds thousands points (comma in US). That is just visual formatting though, they are just a big single value as shown in the second picture.

![excel_2018-03-28_15-23-16](https://user-images.githubusercontent.com/8568089/38035089-e9b7f936-32a3-11e8-81c9-99b113b0a926.png)
Now you can use `=LEFT(A1, 6)` to short the text. Copy and paste them as plain text into a new column to loose the formula and search&replace the dot with a comma. Now you have proper values (not just text).
![excel_2018-03-28_15-27-30](https://user-images.githubusercontent.com/8568089/38035203-284882c4-32a4-11e8-98da-3b1b4422c8be.png)

**Possible Solution:**
Make the output in a proper format by default without the use of a decimal point.
As in only output in seconds or go down a step to milliseconds if accuracy is needed.

As mentioned this might not need a fix, CSV output is useful but I'm not sure if its just a debug feature or will be an actual feature.
Just thought I might mention it. Gave me a bit of trouble to get a final graph.
https://i.imgur.com/zcdOQRo.png

**Comment by Blaast** [comment]
Same happened to me ! My workaround was switching to another country system (like US) and then back to my country's, as I found online.

**Comment by rlguy** [comment]
Hey, thanks for letting me know! The timing values are really only accurate the the millisecond range, so I'll limit those values to 3 decimal places to clean up the text.

I'll add an option to select **US** or **EUR** region format for CSV export.

I am wondering which of the following **EUR** formats import better for you, or if they work at all:

### 1. Commas in numbers (1,23), semicolons as separators

[flip_fluid_stats1.txt](https://github.com/rlguy/Blender-FLIP-Fluids-Beta/files/1857648/flip_fluid_stats1.txt)

```
frame_id;frame_timeline;timestep;substeps;particles_fluid;particles_whitewater;time_mesh_generation;time_velocity_advection;time_fluid_particles;time_pressure_solver;time_whitewater_solver;time_viscosity_solver;time_simulation_objects;time_other;time_total;mesh_surface_enabled;mesh_surface_vertices;mesh_surface_triangles;mesh_surface_bytes;mesh_preview_enabled;mesh_preview_vertices;mesh_preview_triangles;mesh_preview_bytes;mesh_foam_enabled;mesh_foam_vertices;mesh_foam_triangles;mesh_foam_bytes;mesh_bubble_enabled;mesh_bubble_vertices;mesh_bubble_triangles;mesh_bubble_bytes;mesh_spray_enabled;mesh_spray_vertices;mesh_spray_triangles;mesh_spray_bytes;mesh_obstacle_enabled;mesh_obstacle_vertices;mesh_obstacle_triangles;mesh_obstacle_bytes;debug_particles_enabled;debug_particles_vertices;debug_particles_triangles;debug_particles_bytes
0;1;0,042;1;44028;0;0,093;0,000;0,000;0,031;0,000;0,000;0,188;0,016;0,328;TRUE;3886;7768;139856;TRUE;4038;8072;145328;FALSE;0;0;0;FALSE;0;0;0;FALSE;0;0;0;FALSE;0;0;0;TRUE;44028;0;536536
1;2;0,042;1;44028;0;0,109;0,110;0,140;0,078;0,000;0,000;0,000;0,000;0,437;TRUE;3890;7776;140000;TRUE;4040;8076;145400;FALSE;0;0;0;FALSE;0;0;0;FALSE;0;0;0;FALSE;0;0;0;TRUE;44028;0;536536
2;3;0,042;1;44028;0;0,093;0,110;0,141;0,078;0,000;0,000;0,000;0,015;0,438;TRUE;3872;7744;139400;TRUE;4032;8064;145160;FALSE;0;0;0;FALSE;0;0;0;FALSE;0;0;0;FALSE;0;0;0;TRUE;44028;0;536536
3;4;0,042;1;44028;0;0,114;0,114;0,129;0,081;0,000;0,000;0,000;0,000;0,437;TRUE;3956;7908;142376;TRUE;4118;8232;148208;FALSE;0;0;0;FALSE;0;0;0;FALSE;0;0;0;FALSE;0;0;0;TRUE;44028;0;536536
4;5;0,042;1;44028;0;0,120;0,136;0,106;0,075;0,000;0,000;0,000;0,015;0,453;TRUE;4008;8012;144248;TRUE;4170;8336;150080;FALSE;0;0;0;FALSE;0;0;0;FALSE;0;0;0;FALSE;0;0;0;TRUE;44028;0;536536
5;6;0,042;1;44028;0;0,109;0,125;0,140;0,078;0,000;0,000;0,000;0,032;0,484;TRUE;4052;8100;145832;TRUE;4222;8440;151952;FALSE;0;0;0;FALSE;0;0;0;FALSE;0;0;0;FALSE;0;0;0;TRUE;44028;0;536536
6;7;0,042;1;44028;0;0,106;0,121;0,151;0,075;0,000;0,000;0,000;0,000;0,453;TRUE;4080;8156;146840;TRUE;4254;8504;153104;FALSE;0;0;0;FALSE;0;0;0;FALSE;0;0;0;FALSE;0;0;0;TRUE;44028;0;536536
7;8;0,042;1;44028;0;0,094;0,125;0,140;0,078;0,000;0,000;0,000;0,000;0,437;TRUE;4228;8452;152168;TRUE;4390;8776;158000;FALSE;0;0;0;FALSE;0;0;0;FALSE;0;0;0;FALSE;0;0;0;TRUE;44028;0;536536
8;9;0,042;1;44028;0;0,105;0,105;0,136;0,061;0,000;0,000;0,000;0,014;0,422;TRUE;4324;8648;155672;TRUE;4502;9004;162080;FALSE;0;0;0;FALSE;0;0;0;FALSE;0;0;0;FALSE;0;0;0;TRUE;44028;0;536536
```

### 2. Commas in numbers (1,23), commas as separators

[flip_fluid_stats2.txt](https://github.com/rlguy/Blender-FLIP-Fluids-Beta/files/1857655/flip_fluid_stats2.txt)

```
frame_id,frame_timeline,timestep,substeps,particles_fluid,particles_whitewater,time_mesh_generation,time_velocity_advection,time_fluid_particles,time_pressure_solver,time_whitewater_solver,time_viscosity_solver,time_simulation_objects,time_other,time_total,mesh_surface_enabled,mesh_surface_vertices,mesh_surface_triangles,mesh_surface_bytes,mesh_preview_enabled,mesh_preview_vertices,mesh_preview_triangles,mesh_preview_bytes,mesh_foam_enabled,mesh_foam_vertices,mesh_foam_triangles,mesh_foam_bytes,mesh_bubble_enabled,mesh_bubble_vertices,mesh_bubble_triangles,mesh_bubble_bytes,mesh_spray_enabled,mesh_spray_vertices,mesh_spray_triangles,mesh_spray_bytes,mesh_obstacle_enabled,mesh_obstacle_vertices,mesh_obstacle_triangles,mesh_obstacle_bytes,debug_particles_enabled,debug_particles_vertices,debug_particles_triangles,debug_particles_bytes
0,1,"0,042",1,44028,0,"0,093","0,000","0,000","0,031","0,000","0,000","0,188","0,016","0,328",TRUE,3886,7768,139856,TRUE,4038,8072,145328,FALSE,0,0,0,FALSE,0,0,0,FALSE,0,0,0,FALSE,0,0,0,TRUE,44028,0,536536
1,2,"0,042",1,44028,0,"0,109","0,110","0,140","0,078","0,000","0,000","0,000","0,000","0,437",TRUE,3890,7776,140000,TRUE,4040,8076,145400,FALSE,0,0,0,FALSE,0,0,0,FALSE,0,0,0,FALSE,0,0,0,TRUE,44028,0,536536
2,3,"0,042",1,44028,0,"0,093","0,110","0,141","0,078","0,000","0,000","0,000","0,015","0,438",TRUE,3872,7744,139400,TRUE,4032,8064,145160,FALSE,0,0,0,FALSE,0,0,0,FALSE,0,0,0,FALSE,0,0,0,TRUE,44028,0,536536
3,4,"0,042",1,44028,0,"0,114","0,114","0,129","0,081","0,000","0,000","0,000","0,000","0,437",TRUE,3956,7908,142376,TRUE,4118,8232,148208,FALSE,0,0,0,FALSE,0,0,0,FALSE,0,0,0,FALSE,0,0,0,TRUE,44028,0,536536
4,5,"0,042",1,44028,0,"0,120","0,136","0,106","0,075","0,000","0,000","0,000","0,015","0,453",TRUE,4008,8012,144248,TRUE,4170,8336,150080,FALSE,0,0,0,FALSE,0,0,0,FALSE,0,0,0,FALSE,0,0,0,TRUE,44028,0,536536
5,6,"0,042",1,44028,0,"0,109","0,125","0,140","0,078","0,000","0,000","0,000","0,032","0,484",TRUE,4052,8100,145832,TRUE,4222,8440,151952,FALSE,0,0,0,FALSE,0,0,0,FALSE,0,0,0,FALSE,0,0,0,TRUE,44028,0,536536
6,7,"0,042",1,44028,0,"0,106","0,121","0,151","0,075","0,000","0,000","0,000","0,000","0,453",TRUE,4080,8156,146840,TRUE,4254,8504,153104,FALSE,0,0,0,FALSE,0,0,0,FALSE,0,0,0,FALSE,0,0,0,TRUE,44028,0,536536
7,8,"0,042",1,44028,0,"0,094","0,125","0,140","0,078","0,000","0,000","0,000","0,000","0,437",TRUE,4228,8452,152168,TRUE,4390,8776,158000,FALSE,0,0,0,FALSE,0,0,0,FALSE,0,0,0,FALSE,0,0,0,TRUE,44028,0,536536
8,9,"0,042",1,44028,0,"0,105","0,105","0,136","0,061","0,000","0,000","0,000","0,014","0,422",TRUE,4324,8648,155672,TRUE,4502,9004,162080,FALSE,0,0,0,FALSE,0,0,0,FALSE,0,0,0,FALSE,0,0,0,TRUE,44028,0,536536
```

**Comment by Mitsuma** [comment]
Semicolons would be the better choice. Both convert just fine but semicolons makes smaller files due to the lack of extra " ".
Not that it matters in terms of space I suppose but why not.

**Comment by rlguy** [comment]
Great to hear! This will be included in the next version (**0.1.3b**) which should be available by March 31st.

I couldn't think of what to name it either, but it can be easily changed later on if something better comes up. There are tooltips to describe each format:

![untitled](https://user-images.githubusercontent.com/4285191/38060887-8e9f168a-32a1-11e8-956b-9674992c72fc.jpg)
